lib/etag.js

```diff
@@ -0,0 +1,120 @@
+// Load modules
+
+var Fs = require('fs');
+var Crypto = require('crypto');
+var Boom = require('boom');
+var Hoek = require('hoek');
+var LruCache = require('lru-cache');
+
+
+// Declare internals
+
+var internals = {};
+
+
+internals.computeHashed = function (response, stat, callback) {
```

Comment:
Change `callback` to `next` since it is not always called on next tick or if it needs to be consistent, use `Hoek.nextTick()`.